COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016 The Zcash developers
+Copyright (c) 2016-2017 The Zcash developers
 Copyright (c) 2009-2015 The Bitcoin Core developers
```

Comment:
The Bitcoin copyright date should also be updated, because we've merged stuff from upstream.